Backpack/build.gradle

```diff
@@ -1,9 +1,10 @@
 apply plugin: 'com.android.library'
 apply plugin: 'kotlin-android'
 apply plugin: 'org.jetbrains.dokka-android'
+apply plugin: 'com.github.dcendents.android-maven'
```

Comment:
What does this do?

Reply:
Adds publishing plugin. see https://jitpack.io/docs/ANDROID/